ui/load/load.go

```diff
@@ -38,7 +38,7 @@ type Icons struct {
 	ContentAdd, NavigationCheck, NavigationMore, ActionCheckCircle, ActionInfo, NavigationArrowBack,
 	NavigationArrowForward, ActionCheck, ChevronRight, NavigationCancel, NavMoreIcon,
 	ImageBrightness1, ContentClear, DropDownIcon, Cached, ContentRemove, ConcealIcon, RevealIcon,
-	SearchIcon *widget.Icon
+	SearchIcon, ActionSwapHoriz *widget.Icon
```

Comment:
`ActionSwapHoriz` doesn't clear tell what this icon is. I recommend `SwapIcon` or `ExchangeIcon`.